lib/galaxy_test/api/test_workflows.py

```diff
@@ -1134,6 +1134,16 @@ def test_require_unique_step_uuids(self):
         response = self.workflow_populator.create_workflow_response(workflow_dup_uuids)
         self._assert_status_code_is(response, 400)
 
+    def test_require_step_valid_v4_uuids(self):
+        workflow_invalid_uuid = self.workflow_populator.load_workflow(name="test_import")
+        first_step = next(iter(workflow_invalid_uuid["steps"].values()), None)
+        if first_step is None:
+            raise AssertionError("No steps found in the workflow, cannot test invalid UUIDs.")
```

Comment:
A `StopIteration` should be fine for a test?

```suggestion
        first_step = next(iter(workflow_invalid_uuid["steps"].values()))
```

Reply:
Agree! Thanks!